invenio_app_ils/ui/src/invenio_app_ils/common/config.js

```diff
@@ -12,3 +12,5 @@ export const ES_DELAY = 3000;
 export const ES_MAX_SIZE = 1000;
 
 export const SUCCESS_AUTO_DISMISS_SECONDS = 10;
+
+export const DEFAULT_RESULTS_SIZE = 10;
```

Comment:
now that I am thinking to this, we can just expose this config instead of duplicating it:
https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/config.py#L367